src/components/Rocket.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { useDispatch } from 'react-redux';
+import { reserveRocket } from '../redux/rockets/rocketSlice';
+
+const Rocket = (props) => {
+  const { rocketInfo } = props;
+  const {
+    id, name, description, image, reserved,
+  } = rocketInfo;
+  const dispatch = useDispatch();
+
+  const onClick = () => {
+    dispatch(reserveRocket(id));
+  };
+
+  return (
+    <section className="rocket flex-row">
+      <img src={image} alt={name} className="rocket__img" />
+      <div className="rocket__info flex-column">
+        <h4 className="rocket__heading">{name}</h4>
+        <p data-testid="msn" className="rocket__description">
+          {reserved && <span>Reserved</span>}
+          {description}
+        </p>
+        <button data-testid="btn" type="button" className={`rocket__reserve ${reserved && 'rocket__cancel'}`} onClick={onClick}>{reserved ? 'Cancel Reservation' : 'Reserve'}</button>
```

Comment:
-  Please use React conditional rendering syntax for conditions instead of the Ternary operator as this is part of the requirements.
You can use the syntax:
`{ reserved && ( // code for reserved rocket)}
{ !reserved && ( // code for not reserved rocket)}`

You can consider the below code for the button. 
``` ruby
<button data-testid="btn" type="button" className={`rocket__reserve ${reserved && 'rocket__cancel'}`} onClick={onClick}>
          {reserved && <span>Cancel Reservation</span>}
          {!reserved && <span>Reserve</span>}
        </button>